src/Data/Models/WalletWithdrawalRequest.cs

```diff
@@ -57,7 +57,12 @@ public enum WalletWithdrawalRequestStatus
         /// <summary>
         /// Marked when a error happens when broadcasting the TX
         /// </summary>
-        Failed = 6
+        Failed = 6,
+
+        /// <summary>
+        /// The PSBT is being signed by NodeGuard
+        /// </summary>
+        FinalizingPSBT = 7
```

Comment:
```suggestion
        /// <summary>
        /// The PSBT is being signed by NodeGuard after all human required signatures have been collected
        /// </summary>
        FinalizingPSBT = 7
```

So this is basically, all human cosigners did they work, now its time for NG to sign the funded PSBT which has been validated by LND.